README.md

```diff
@@ -343,6 +343,7 @@ Table of Contents
   * [Gremlin](https://www.gremlin.com/gremlin-free-software) — Gremlin's Chaos Engineering tools allow you to safely, securely, and simply inject failure into your systems to find weaknesses before they cause customer-facing issues. Gremlin Free provides access to Shutdown and CPU attacks on up to 5 hosts or containers.
   * [gridlastic.com](https://www.gridlastic.com/) — Selenium Grid testing with free plan up to 4 simultaneous selenium nodes/10 grid starts/4,000 test minutes/month
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  10 team members, Demo app and unlimited projects, 5,000 snapshots / month.
+  * [reflect.run](https://reflect.run) - Codeless automated tests for web apps. Tests can be scheduled in-app or executed from a CI/CD tool.  Each test run includes a full video recording along with console and network logs.
```

Comment:
thanks, please add a bit about the free tier limits - see other entries here like "0 team members, Demo app and unlimited projects, 5,000 snapshots / month."

Reply:
👍 I added this sentence:

> The free tier includes an unlimited number of saved tests, with 25 test runs per month and up to 3 users.